lms/djangoapps/grades/models.py

```diff
@@ -636,3 +636,28 @@ def get_override(cls, user_id, usage_key):
 def prefetch(user, course_key):
     PersistentSubsectionGradeOverride.prefetch(user.id, course_key)
     VisibleBlocks.bulk_read(user.id, course_key)
+
+
+class PersistentSubsectionGradeOverrideHistory(models.Model):
+    """
+    A django model tracking persistent grades override audit records.
+    """
+    PROCTORING = 'PROCTOR'
+    GRADEBOOK = 'GRADEBK'
+    OVERRIDE_FEATURES = (
+        (PROCTORING, 'Proctoring')
+        (GRADEBOOK, 'Gradebook')
+    )
+
+    class Meta(object):
+        app_label = "grades"
+
+    override = models.ForeignKey(PersistentSubsectionGradeOverride, related_name='override_history')
+    feature = models.CharField(
+        max_length=8,
+        choices=OVERRIDE_FEATURES,
+        default=PROCTORING
+    )
+    user_id = models.IntegerField(blank=True)
```

Comment:
You could make this a ForeignKey to `auth.User`.  That will make it easier to deal with saying which user modified an override in the API code.